pulsar-flink-connector/src/main/java/org/apache/flink/streaming/connectors/pulsar/internal/PulsarClientUtils.java

```diff
@@ -62,6 +62,7 @@ public static ClientConfigurationData newClientConf(String serviceUrl, Propertie
             clientConf.setAuthParams(properties.getProperty(PulsarOptions.AUTH_PARAMS_KEY));
             clientConf.setAuthPluginClassName(
                     properties.getProperty(PulsarOptions.AUTH_PLUGIN_CLASSNAME_KEY));
+            clientConf.setTlsTrustCertsFilePath(PulsarOptions.TLS_TRUSTCERTS_FILEPATH);
```

Comment:
This could be a mistake? Have you tested on your local machine?

Reply:
> This could be a mistake? Have you tested on your local machine?

Sorry,I didn't catch your meaning.  I had tested on our pulsar cluster. PS: our cluster is run with both token authentication and tls for transport encryption.  If only config one auth-plugin, connector would get error.